src/themes/default/components/core/blocks/Microcart/Microcart.vue

```diff
@@ -184,6 +189,20 @@ export default {
     },
     onEscapePress () {
       this.closeMicrocart()
+    },
+    clearCart () {
+      this.$store.dispatch('notification/spawnNotification', {
+        type: 'warning',
+        item: this.product,
```

Comment:
```suggestion
```
i think this shouldn't be here

Reply:
I thought I removed it 😿 
Thanks for spotting.